sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/files/amd64_defconfig-6.6

```diff
@@ -70,6 +70,7 @@ CONFIG_ISCSI_IBFT=y
 CONFIG_ISCSI_IBFT_FIND=y
 CONFIG_ITCO_VENDOR_SUPPORT=y
 CONFIG_ITCO_WDT=m
+CONFIG_IGC=m
```

Comment:
I was thinking that this driver might be also useful for ARM64. But from the docs, this driver seems to be used mostly on the NUCs and not the ARM64 servers (ARM64 boards do not use Intel as well).